crates/wash-cli/src/common/get_cmd.rs

```diff
@@ -30,10 +44,78 @@ pub async fn handle_command(command: GetCommand, output_kind: OutputKind) -> Res
             } else {
                 sp.update_spinner_message(" Retrieving hosts for inventory query ...".to_string());
             }
-            let invs = get_host_inventories(cmd).await?;
-            get_host_inventories_output(invs)
+            get_inventory_handler(cmd, sp).await?
         }
     };
 
     Ok(out)
 }
+
+async fn get_inventory_handler(
+    cmd: GetHostInventoriesCommand,
+    sp: Spinner,
+) -> Result<CommandOutput> {
+    if cmd.watch {
```

Comment:
We should make sure that the watch loop has a configurable time (i.e. how long between each inventory call). That could either be done by making this an option with custom parsing to provide a default or adding another flag that lets you set the time

Reply:
Done ! :saluting_face: 
I've set the input to milis right now, should it be seconds or is it fine ?

Reply:
Milis is fine!